zebra-chain/src/transaction/serialize.rs

```diff
@@ -309,8 +312,6 @@ impl ZcashDeserialize for Option<orchard::ShieldedData> {
         let actions: Vec<orchard::Action> = (&mut reader).zcash_deserialize_into()?;
 
         if actions.is_empty() {
```

Comment:
Do you think we should summarise the spec here as well?

Reply:
This is ok and i added it in https://github.com/ZcashFoundation/zebra/pull/2116/commits/bf2182941f3686a45b612e2014a92b46c314f8fb but replaced the `…` with `...`.